### ISSUE TITLE
conflicting requirements cause pip-compile to fail

### ISSUE THREAD

**Issue by chandeeland** [intuit/oauth-pythonclient#2]
all builds of intuit-oauth (1.1.0 and 1.0.2) will fail when attempting to use `pip-compile`

```
Could not find a version that matches coverage==4.0.3,>=4.4
Tried: 3.0, 3.0.1, 3.1, 3.2, 3.3, 3.3.1, 3.4, 3.5, 3.5.1, 3.5.2, 3.5.3, 3.6, 3.7, 3.7.1, 4.0, 4.0, 4.0.1, 4.0.1, 4.0.2, 4.0.2, 4.0.3, 4.0.3, 4.1, 4.1, 4.2, 4.2, 4.3, 4.3.1, 4.3.2, 4.3.3, 4.3.4, 4.4, 4.4.1, 4.4.2, 4.5, 4.5.1, 4.5.2
Skipped pre-versions: 3.0b3, 3.1b1, 3.2b1, 3.2b2, 3.2b3, 3.2b4, 3.4b1, 3.4b2, 3.5b1, 3.5.1b1, 3.5.2b1, 3.6b1, 3.6b2, 3.6b3, 4.0a1, 4.0a2, 4.0a3, 4.0a4, 4.0a5, 4.0a5, 4.0a5, 4.0a6, 4.0a6, 4.0a6, 4.0b1, 4.0b1, 4.0b2, 4.0b2, 4.0b3, 4.0b3, 4.1b1, 4.1b1, 4.1b2, 4.1b2, 4.1b3, 4.1b3, 4.2b1, 4.2b1, 4.4b1, 5.0a1, 5.0a2, 5.0a3, 5.0a4
There are incompatible versions in the resolved dependencies.
```

in `requirements.txt` intuit-oauth specifies `python-coveralls` and `coverage >=4.4`

however python-coveralls (v2.9.1 from 2017) requires `coverage==4.0.3`
in it's [setup.py](https://github.com/z4r/python-coveralls/blob/master/setup.py) line 25

it is of course impossible to have both coverage>=4.4 and coverage==4.0.3

please adjust your requirements.txt

**Comment by skyarkitekten** [comment]
i have the same issue as @chandeeland. Between this and 2 open issues with the node client libraries, I am becoming increasingly frustrated with Intuit's ability to ship quality SDK's

**Comment by nbhambhani** [comment]
Hi @chandeeland and @skyarkitekten thank you for reporting this issue.
Library `pytest-cov>=2.6.0` required `coverage>=4.4`, and as you pointed out `python-coveralls` requires `coverage==4.0.3`.
For now, I have [changed](https://github.com/intuit/oauth-pythonclient/commit/2b1646501adbde8f53a3ec25eb224872611d362b) `pytest-cov` to use version `2.5.0`. I released the package under version `1.1.1`. Please let me know if this fix doesn't work for you or if you find other issues. We welcome all feedback and PRs.
